src/main.rs

```diff
@@ -6,10 +6,11 @@
 
 use crate::compile::CompileStruct;
 use crate::link::LinkedProgram;
-use crate::pos::try_display_location;
+//use crate::pos::try_display_location;
```

Comment:
If this use isn't needed anymore it should be deleted rather than commented out.

Reply:
Deleted!